posts/2024-06-20_myg-9/README.md

```diff
@@ -0,0 +1,255 @@
+- PR: https://github.com/gnolang/gno/pull/2225
+Issue: https://github.com/gnolang/gno/issues/2193
+Des: I propose integrating certain utility smart contracts from Ethereum into Gnoland. Now i'm working on defining the Bitmap, NonceManager and Queue packages, which can provide essential functionality for the Gnoland ecosystem.
+Status: Waiting for reviewing
+- PR: https://github.com/gnolang/gno/pull/2234
+Issue: https://github.com/gnolang/gno/issues/2231
+Des: We should either implement this, or remove the flag until the functionality is implemented. Same goes for the –prove flag.
+Status: Merged
+
+- PR: 
+    - Support extensions like Metadata, RoyaltyInfo for GRC721  https://github.com/gnolang/gno/pull/1962 (Merged)
+    - Deprecate & remove std.CurrentRealmPath() https://github.com/gnolang/gno/pull/2087 (Reviewing)
+    - limit package path length https://github.com/gnolang/gno/pull/2108 (Aprroved)
+    - Implement Bitmap package https://github.com/gnolang/gno/pull/2115 (Need review)
+    - Implement Nonces package https://github.com/gnolang/gno/pull/2123 (Need review)
+    - GasUsed() for GnoVM std https://github.com/gnolang/gno/pull/2149
+- Issues:
+    - Panic when getting keypair information https://github.com/gnolang/gno/issues/2133
+    - Proposal: Integrate Sponsor Mechanism for Transaction Fees https://github.com/gnolang/gno/issues/2152
+
+## Student Contributor Program
+
+**Mustapha**
+* Made a V0 Auction dapp ([PR#2265](https://github.com/gnolang/gno/pull/2265)) 
+
+**Antonio**
+- GNOWLEDGE
+A [realm](https://github.com/iam-agf/Gnowledge) to simulate a Stackoverflow styled. Sharing to get some feedback. You can interact with it via https://github.com/iam-agf/Gnowledge-website .
+
+**Antonio**
+* https://github.com/iam-agf/Shiken 
+
+# New Content
+
+- [Key/Value Stores: How We Improved the Performance of Our tx-indexer by 10x](https://gno.land/r/gnoland/blog:p/kv-stores-indexer)
+- [Introducing Gno Studio, the Premier Builder Suite for Gno.land](https://gno.land/r/gnoland/blog:p/gno-studio-intro)
+- [Introducing our new Gno.land logo: the gnome](https://gno.land/r/gnoland/blog:p/the-gnome)
+- [Gnomes Spotted in Belgrade, Serbia: Recap from the Engineering Retreat and Golang Serbia Meetup](https://gno.land/r/gnoland/blog:p/gnomes-in-serbia)
+- [Test4 Explained](https://gno.land/r/gnoland/blog:p/test4-explained)
+
+# Events and Meetups
+
+## Past events
+
+- GoLang Serbia Meetup / Belgrade / May 23. We used the Gno core team's retreat in Belgrade to connect with the local Go developers and possible contributors over the next few months to build the ecosystem. [Full recap](https://gno.land/r/gnoland/blog:p/gnomes-in-serbia).
+- We're currently wrapping up **GopherCon EU** in Berlin, expect an update soon!
+
+## Upcoming events
+
+- **GopherCon US** / Chicago / July 7th - 10th
+- **Nebular Summit** / Brussels / July 12th/13th
+
+If you plan to attend any of these events, find us at our booth or on the hallway track, and let's talk!
```

Comment:
We could also add the weekly Developer Office Hours on Discord as the final CTA...